source/scripts/utils.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+const fs = require('fs')
+const dataDir = path.resolve(__dirname, '../data/')
+
+exports.createDataDir = () => {
+	if (!fs.existsSync(dataDir)) {
+		fs.mkdirSync(dataDir)
+	}
+}
+
+exports.writeInDataDir = (filename, data) => {
+	fs.writeFileSync(path.join(dataDir, filename), JSON.stringify(data))
```

Comment:
```suggestion
	fs.writeFileSync(path.join(dataDir, filename), JSON.stringify(data, null, 2))
```

Ça permet de formater le fichier avec l'indentation ça sera plus facile à deboguer dans le futur.